### PR TITLE
Remove infection monkey imports from plugins

### PR DESCRIPTION
**Pull request by ilija-lazoroski**
# What does this PR do?

Add any further explanations here.

## PR Checklist
* [ ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [ ] Do all unit tests pass?
* [ ] Do all end-to-end tests pass?
* [ ] Any other testing performed?
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [ ] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

**Comment by cakekoa** [comment]
What's the motivation behind this? get_agent_id will prevent you from splitting out the plugins?

**Comment by ilija-lazoroski** [comment]
> What's the motivation behind this? get_agent_id will prevent you from splitting out the plugins?

Yep, it is defined in the agents and only used in the agent and these three tests. I think that the usage in the tests is not needed.

**Review by cakekoa — APPROVED** [review]

File: monkey/tests/unit_tests/agent_plugins/exploiters/hadoop/test_hadoop_exploiter.py [hunk 1]
```diff
@@ -17,11 +18,16 @@
...
+    AgentID,
+    NetworkPort,
+    NetworkProtocol,
+    NetworkService,
+    OperatingSystem,
+    PortStatus,
+)
 
-AGENT_ID = get_agent_id()
+AGENT_ID: AgentID = uuid4()
```
Comment (mssalvatore) [inline monkey/tests/unit_tests/agent_plugins/exploiters/hadoop/test_hadoop_exploiter.py:30]: Can we do this?

```suggestion
AGENT_ID = AgentID()
```

Comment (ilija-lazoroski) [reply monkey/tests/unit_tests/agent_plugins/exploiters/hadoop/test_hadoop_exploiter.py]: Nope. Tried it for zerologon.

File: monkey/tests/unit_tests/agent_plugins/exploiters/zerologon/test_zerologon.py [hunk 2]
```diff
@@ -32,7 +31,7 @@ def mock_report_login_attempt(**kwargs):
     monkeypatch.setattr(obj, "report_login_attempt", mock_report_login_attempt)
     monkeypatch.setattr(obj, "host", TargetHost(ip=IPv4Address("1.1.1.1")))
     monkeypatch.setattr(obj, "agent_event_publisher", mock_agent_event_publisher)
-    monkeypatch.setattr(obj, "agent_id", get_agent_id())
+    monkeypatch.setattr(obj, "agent_id", uuid4())
```
Comment (mssalvatore) [inline monkey/tests/unit_tests/agent_plugins/exploiters/zerologon/test_zerologon.py:34]:
```suggestion
    monkeypatch.setattr(obj, "agent_id", AgentID())
```

**Review by mssalvatore — APPROVED** [review]

**Pull request merged**